ansible/roles/multifaucet/tasks/main.yml

```diff
@@ -45,24 +45,32 @@
 
 - name: Create promo codes table
   community.docker.docker_container_exec:
-    container: multifaucet_db_1
+    container: multifaucet-db-1
     command: >
       mysql -h localhost -D multifaucet -u multifaucet --password="multifaucet"
       -e 'CREATE TABLE IF NOT EXISTS faucet_promo_codes (id int NOT NULL, code varchar(255), minimum_payout int, maximum_payout int, PRIMARY KEY (id));'
 
 - name: Load MN promo code
   community.docker.docker_container_exec:
-    container: multifaucet_db_1
+    container: multifaucet-db-1
     command: >
       mysql -h localhost -D multifaucet -u multifaucet --password="multifaucet"
```

Comment:
You can pass all three rows after VALUES instead of calling client 3 times

Reply:
One of them only applies to devnets, while it's wasting a few CPU cycles it won't actually have any effect since there's no actual traffic at this point in setup